windows/src/developer/inst/kmdev.wxs

```diff
@@ -256,7 +256,7 @@
       <!-- Redistributable files -->
 
       <Component Directory="dirRedistSetup">
-        <File Name="setup.exe" Source="..\..\..\bin\desktop\setup.exe" KeyPath="yes" />
+        <File Name="setup.exe" Source="..\..\..\bin\desktop\setup-redist.exe" KeyPath="yes" />
```

Comment:
I take it `Name` doesn't need to change to "setup-redist.exe"

Reply:
Correct. In theory, we could do that, but it means more changes to other locations in the build.